### ISSUE TITLE
Accessibility: ARIA Label Missing on 7Z Compression  Dialogue Pop-up (Windows)

### ISSUE THREAD

**Issue by Murcielago7384** [ip7z/7zip#246]
When using a screen-reader, there is a missing ARIA label for one of the buttons on the 7Z compression pop-up that is accessed via Windows File Explorer context menu. The SR announces it as 'button' so it is impossible to know the functionality without activating it.

I am unsure if any other buttons are missing ARIA labels in 7Z, though I'd be very grateful if this could be investigated.

** Steps to reproduce **
1. Make sure you have a screen-reader running on your Windows system. NVDA is a free and open source SR available from NV Access.
2. Go into File Explorer and find a folder you wish to compress.
3. Hit Shift+F10 to bring up the context menu and find "7z > add to archive".
4. The button next to the file name field is the one that is missing an ARIA label. You will notice that NVDA announces only "button".

**Comment by sov44** [comment]
Another bug
If you press Shift+F10 on a file (for example, a text file) in Explorer, then many unnecessary commands appear in the 7-zip context menu, including unpack this file.
